From 154cc8632e1b3cc002fc77d9dccb0359462132aa Mon Sep 17 00:00:00 2001
From: cheney <11722200@qq.com>
Date: Tue, 18 Mar 2025 17:56:59 +0800
Subject: [PATCH 250/533] Changelogs: Merge the latest TAG(EIC7X-2025.02) code,
 update macro definitions to align with upstream changes, and modify relevant
 files for FML13V03 adaptation.

---
 .../boot/dts/eswin/eic7702-deepcomputing-fml13v03.dts    | 6 +++---
 arch/riscv/configs/fml13v03_defconfig                    | 9 ++++++---
 2 files changed, 9 insertions(+), 6 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03.dts b/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03.dts
index cb60d6d97362..a4f30151ca9b 100644
--- a/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03.dts
+++ b/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03.dts
@@ -708,7 +708,7 @@ &d0_i2s1 {
     status = "okay";
     numa-node-id = <0>;
     #sound-dai-cells = <0>;
-    clocks = <&d0_clock WIN2030_CLK_VO_I2S_MCLK>, <&d0_clock WIN2030_APLL_FOUT1>;
+    clocks = <&d0_clock EIC7700_CLK_VO_I2S_MCLK>, <&d0_clock EIC7700_APLL_FOUT1>;
     clock-names = "mclk", "apll";
     pinctrl-names = "default";
     pinctrl-0 = <&pinctrl_die0_i2s1_default>;
@@ -1207,7 +1207,7 @@ d0_es8326_0: es8326@19 {
 		reg = <0x19>;
 		status = "okay";
 
-        clocks = <&d0_clock WIN2030_CLK_VO_I2S_MCLK>;
+        clocks = <&d0_clock EIC7700_CLK_VO_I2S_MCLK>;
         clock-names = "mclk";
 
 		interrupt-parent = <&porta>;
@@ -1467,7 +1467,7 @@ &pwm0 {
 &d1_pwm0 {
     status = "okay";
     clock-names = "pclk";
-    clocks = <&d1_clock WIN2030_CLK_LSP_TIMER_PCLK>;
+    clocks = <&d1_clock EIC7700_CLK_LSP_TIMER_PCLK>;
     pinctrl-names = "default";
     pinctrl-0 = <&d1_pinctrl_pwm0_default>;
 };
diff --git a/arch/riscv/configs/fml13v03_defconfig b/arch/riscv/configs/fml13v03_defconfig
index 87f975b2cdce..1c3e14d9a251 100755
--- a/arch/riscv/configs/fml13v03_defconfig
+++ b/arch/riscv/configs/fml13v03_defconfig
@@ -1,3 +1,5 @@
+CONFIG_LOCALVERSION="-eic7702"
+# CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
 CONFIG_WATCH_QUEUE=y
@@ -29,7 +31,7 @@ CONFIG_BLK_DEV_INITRD=y
 CONFIG_EXPERT=y
 # CONFIG_SYSFS_SYSCALL is not set
 CONFIG_PERF_EVENTS=y
-CONFIG_SOC_SIFIVE=y
+CONFIG_SOC_ESWIN=y
 CONFIG_SOC_VIRT=y
 CONFIG_SMP=y
 CONFIG_SCHED_CLUSTER=y
@@ -483,6 +485,7 @@ CONFIG_DW_WATCHDOG=y
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_TPS549D22=y
+CONFIG_REGULATOR_ES5340=y
 CONFIG_REGULATOR_PCA9450=y
 CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_SUPPORT_FILTER=y
@@ -764,7 +767,7 @@ CONFIG_CROS_EC_SPI=m
 # CONFIG_CROS_EC_SENSORHUB is not set
 # CONFIG_CROS_EC_SYSFS is not set
 # CONFIG_CROS_USBPD_NOTIFY is not set
-CONFIG_COMMON_CLK_WIN2030=y
+CONFIG_COMMON_CLK_EIC7700=y
 CONFIG_TIMER_ESWIN=y
 CONFIG_MAILBOX=y
 CONFIG_ESWIN_MBOX=y
@@ -787,7 +790,7 @@ CONFIG_ESWIN_IOMMU_RSV=y
 CONFIG_ESWIN_DMA_MEMCP=y
 CONFIG_PWM=y
 CONFIG_PWM_ESWIN=y
-CONFIG_RESET_ESWIN_WIN2030=y
+CONFIG_RESET_ESWIN=y
 CONFIG_INTERCONNECT=y
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_POSIX_ACL=y
-- 
2.52.0

